lib/array_intersection.rb

```diff
@@ -1,3 +1,11 @@
 def intersection(list1, list2)
-  raise NotImplementedError, "Intersection not implemented"
+  intersect = []
+  list1.each do |num|
+    list2.each do |num2|
+      if num == num2
+        intersect << num
+      end
+    end
+  end
+  return intersect
```

Comment:
This works, but it's O(n * m) in time complexity where n and m are the lengths of the arrays.  There is an O(n + m) solution using a hash which is **much** better.  

Think about how you could make this better with a hash.